pkg/webhook/useridentity/mutatingWebhook.go

```diff
@@ -87,6 +86,7 @@ func (a *AdmissionHandler) serve(w io.Writer, r *http.Request, admit admitFunc)
 
 func (a *AdmissionHandler) mutateResource(ar v1.AdmissionReview) *v1.AdmissionResponse {
 	klog.V(2).Info("mutating custom resource")
```

Comment:
We always use v(4) here

Reply:
Thanks Quijian. New commit has the requested changes. Not really sure where to put the "contains" functions - it probably belongs in the util package but couldn't find a good home for it.

Reply:
there is a ContainsString in util pkg.